connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -2469,6 +2447,93 @@ ble_error_t Gap::startAdvertising(
 }
 #endif
 
+#if BLE_FEATURE_EXTENDED_ADVERTISING
+void Gap::start_advertising_enable(
```

Comment:
The name of the function doesn't reflect what it does: queuing a request to start a new advertising set.

Reply:
will rename